sources/packages/api/src/database/entities/msfaa-number.model.ts

```diff
@@ -77,4 +77,13 @@ export class MSFAANumber extends RecordDataModel {
     referencedColumnName: ColumnNames.ID,
   })
   student: Student;
+  /**
+   * Application that creates the MSFAA Number.
+   */
+  @ManyToOne(() => Application, { eager: false, cascade: true })
+  @JoinColumn({
+    name: "reference_application_id",
+    referencedColumnName: ColumnNames.ID,
+  })
+  referenceApplication: Application;
```

Comment:
Should it be marked as optional with "_referenceApplication?_"?

Reply:
We agreed to have it as mandatory field, as this should be available when we create a application that generated a New MSFAA Number.

Reply:
Agree, it must be mandatory, but on the migrations it was optional. So, as we talked, we are going to be changing the migrations in an upcoming ticket to the column be mandatory on DB also.